src/LordRingsApi.ts

```diff
@@ -0,0 +1,96 @@
+import axios, { AxiosInstance } from 'axios';
+import {IQuote} from './models/interfaces/IQuote';
+import {IMovie} from './models/interfaces/IMovie';
+/*
+    Here is my wrapper around the API 
+    I feel like it is the best wrapper anyone has ever wrapped
+    Should prolly hire me...
+*/
+class LordRingsApi {
+
+    private client: AxiosInstance; //I would of made some dependency injection for things like this but eh I dont have enough time
+
+    constructor(baseURL: string) { //Also would of liked the baseURL to be controlled by config and dependency injection with the ability to overide in the calling application in case there was ever a need to 
```

Comment:
```suggestion
    constructor(baseURL: string) { //Also would have liked the baseURL to be controlled by config and dependency injection with the ability to overide in the calling application in case there was ever a need to 
```